src/legacy/core_plugins/markdown_vis/public/markdown_fn.js

```diff
@@ -21,7 +21,7 @@ import { functionsRegistry } from 'plugins/interpreter/registries';
 import { i18n } from '@kbn/i18n';
 
 export const kibanaMarkdown = () => ({
-  name: 'kibana_markdown',
+  name: 'markdownvis',
```

Comment:
nit: should we use camelCase? `markdownVis`

Reply:
Just a reminder, the naming is internal, though it does show up however you enter it if you auto-generate docs or auto-complete or whatever. `markdownvis` and `markdownVis` will do the same in the expression, but the later looks better and is easier to read IMO.